### PR TITLE
Fix more CS whitespace

### PR DESCRIPTION
**Pull request by mvorisek**

### CODE REVIEW

**Comment by alecpl** [comment]
I don't like it. Also, I think it was a mistake to include `@PhpCsFixer` rules from the start. We should have started with `@PSR1` + `@PSR12` and then introduce more strict rules.

**Comment by mvorisek** [comment]
What part do you not like?

no useless else - this improves code by reducing indentation, this is natural and mostly done already, newly, the remaining code is fixed and asserted by CI

blank line before statement - I know you do no like it much, so I skipped this for `continue` and `break` but for `return`, `throw` etc. it is good and make again the code more readable, as these are code flow exiting/returning statements. Again, mostly done naturally, now remaining code is fixed and asserted by CI.

> Also, I think it was a mistake to include `@PhpCsFixer` rules from the start. We should have started with `@PSR1` + `@PSR12` and then introduce more strict rules.

Why? `@PhpCsFixer` is "optimal ruleset" in sense of CS strictness and PHP CS Fixer possibilities. Of course, starting with this ruleset will be wrong, that is why I added it, but disabled all the rules, so be we land the changes in reviewable parts. So effectively, we do it this way.

File: program/lib/Roundcube/rcube_addressbook.php [hunk 1]
```diff
@@ -688,6 +688,7 @@ public static function compose_list_name($contact)
                     $result[] = $contact[$field];
                 }
             }
+
```
Comment (alecpl) [inline program/lib/Roundcube/rcube_addressbook.php:691]: You see, here it makes sense, i.e. after "}", but not in other places. We don't have to be that strict. I see no value in being that strict.

File: program/lib/Roundcube/rcube_config.php [hunk 2]
```diff
@@ -576,12 +577,12 @@ public function header_delimiter()
             $delim = $this->prop['mail_header_delimiter'];
             if ($delim == "\n" || $delim == "\r\n") {
                 return $delim;
-            } else {
```
Comment (alecpl) [inline program/lib/Roundcube/rcube_config.php:579]: no_useless_else=true is all right. I prefer that too.

File: program/lib/Roundcube/rcube_addressbook.php (hunk 1 above)
Comment (mvorisek) [reply program/lib/Roundcube/rcube_addressbook.php:691]: Done - I patched BlankLineBeforeStatement fixer and applied it.

**Pull request merged**